win32/tests/reals_win32_ut/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+set(theseTestsName reals_win32_ut)
```

Comment:
>reals_win32_ut [](start = 19, length = 14)

Lets do the same thing for reals_linux_ut, because there are bugs there that would have been caught by that test :)